src/Features/LanguageServer/Protocol/LanguageServerTarget.cs

```diff
@@ -76,6 +76,52 @@ internal class LanguageServerTarget : ILanguageServerTarget
                 supportedLanguages,
                 serverKind);
             Queue.RequestServerShutdown += RequestExecutionQueue_Errored;
+
+            foreach (var metadata in RequestDispatcher.GetRegisteredMethods())
```

Comment:
this is where we register the method + request and response types with streamjsonrpc.